backends/backends.c

```diff
@@ -1069,7 +1077,7 @@ void *backends_main(void *ptr) {
                                     "Content-Length: %zu\r\n"
                                     "Content-Type: application/x-www-form-urlencoded\r\n\r\n",
                                     remote_write_path,
-                                    hostname,
+                                    remote_hostname,
```

Comment:
Please use the `destination`. It conforms to https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html.